<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountAdmin</class>
 <widget class="QDialog" name="AccountAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#AccountAdmin{
border-image:url(:/Image/登录界面.png);

}



</string>
  </property>
  <widget class="QPushButton" name="pbt_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(116, 224, 191);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>撤销修改账户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(116, 224, 191);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(254, 254, 254, 150);
border-radius:10%;
padding-left:20%;
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(116, 224, 191);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>删除账户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>提交修改账户</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_account">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(244,244,244,200);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>显示全部账户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>添加账户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>电话号码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(116, 224, 191);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_dishes">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(244,244,244,200);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_8">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>390</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>提交修改菜品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(116, 224, 191);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>撤销修改菜品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_10">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>430</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>添加菜品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_11">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>430</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(116, 224, 191);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>删除菜品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_12">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>390</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>显示全部菜品</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>菜名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(254, 254, 254, 150);
border-radius:10%;
padding-left:20%;
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_13">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>350</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(116, 224, 191);
border-radius:10%;
color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pbt_5</tabstop>
  <tabstop>pbt_4</tabstop>
  <tabstop>pbt_1</tabstop>
  <tabstop>tableView_account</tabstop>
  <tabstop>pbt_6</tabstop>
  <tabstop>pbt_3</tabstop>
  <tabstop>pbt_2</tabstop>
  <tabstop>pbt_7</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
